tooling/SecretManager.VisualStudio/ProjectLocalSecretsManager.cs

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.VisualStudio.ProjectSystem.Properties;
+using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Threading;
+using Task = System.Threading.Tasks.Task;
+
+namespace Microsoft.Extensions.SecretManager.VisualStudio
```

Comment:
Not sure if naming was dicussed yet, but normally something that provides visual studio integration would be named like `Microsoft.VisualStudio...`.

Reply:
hasn't been discussed. What about `Microsoft.VisualStudio.Extensions.SecretManager`?

Reply:
I would drop `Extensions` since it means something else in the VS context.